server/common/utils/helpers.test.js

```diff
@@ -400,3 +403,20 @@ describe('function responseWithComment', () => {
     );
   });
 });
+
+describe('function responseWithCohortDetails', () => {
+  const { _id: userId } = usersMock[0];
+  const result = responseWithCohortDetails(cohortDetailsMock, userId);
+
+  it('returns cohorts details with desired properties', () => {
```

Comment:
braces not necessary

Reply:
I can't remove braces because `it`  must return Promise or undefined